tests/baselines/reference/accessorField5(target=es2015).js

```diff
@@ -0,0 +1,53 @@
+//// [accessorField5.ts]
+class C1 {
+    accessor ["w"]: any;
+    accessor ["x"] = 1;
+    static accessor ["y"]: any;
+    static accessor ["z"] = 2;
+}
+
+declare var f: any;
+class C2 {
+    // @ts-ignore
```

Comment:
`A computed property name in a class property declaration must have a simple literal type or a 'unique symbol' type.`. This is legal JavaScript however, we just issue an error since we can't reliably determine the shape of the class.